1-exercises/A-reading-list/script.js

```diff
@@ -1,27 +1,41 @@
 function readingList(books) {
   // Write your code here...
+  let newUl = document.createElement("ul");
+  document.getElementById("content").append(newUl);
+  for (let book of books) {
+    let newParagraph = document.createElement("p");
+    newParagraph.innerText = `${book.title} by ${book.author}`;
+    let newImg = document.createElement("img");
+    newImg.src = `${book.bookCoverImage}`;
+    let newLi = document.createElement("li");
+    book.alreadyRead
+      ? newLi.classList.add("green")
+      : newLi.classList.add("red");
+    newLi.append(newParagraph, newImg);
+    newUl.append(newLi);
+  }
```

Comment:
Well done here, I appreciate the use of native DOM manipulation APIs rather than using innerHTML or equivalent methods.

Reply:
I read somewhere that innerHTML is dangerous security wise, so I'm trying to avoid it as best as possible even if it isn't on an input.